client/rest-high-level/src/main/java/org/elasticsearch/client/IndexLifecycleRequestConverters.java

```diff
@@ -78,7 +78,7 @@ static Request deleteLifecyclePolicy(DeleteLifecyclePolicyRequest deleteLifecycl
     static Request removeIndexLifecyclePolicy(RemoveIndexLifecyclePolicyRequest removePolicyRequest) {
         String[] indices = removePolicyRequest.indices() == null ?
                 Strings.EMPTY_ARRAY : removePolicyRequest.indices().toArray(new String[] {});
-        Request request = new Request(HttpDelete.METHOD_NAME,
+        Request request = new Request(HttpPost.METHOD_NAME,
```

Comment:
You'll also need to change this in [the associated test](https://github.com/elastic/elasticsearch/blob/216c761a5dd81e65ca9826fb0793ddc53cd2ff27/client/rest-high-level/src/test/java/org/elasticsearch/client/IndexLifecycleRequestConvertersTests.java#L102)

Reply:
damn. test fail. I ran

```
./gradlew ':client:rest-high-level:test' -Dtests.class=org.elasticsearch.client.indexlifecycle*
```

which doesn't include those! thanks @gwbrown